opentasks/src/main/java/org/dmfs/tasks/utils/TaskText.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2017 dmfs GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dmfs.tasks.utils;
+
+/**
+ * A {@link CharSequence} that describes one or more property of a task in some format.
+ * <p>
+ * (E.g.: The text for sharing a task details for other apps.)
+ *
+ * @author Gabor Keszthelyi
+ */
+public interface TaskText extends CharSequence
```

Comment:
This interface looks a bit like over-engineering to me, especially since you never use it for a parameter or variable.

I suggest to get rid of it.